apps/web/pages/api/orgMigration/removeTeamFromOrg.ts

```diff
@@ -0,0 +1,63 @@
+import { getFormSchema } from "@pages/settings/admin/orgMigrations/removeTeamFromOrg";
```

Comment:
Moved this code from website as is. Could think about moving these endpoints to tRPC later.